plugins/techdocs-backend/src/plugin.ts

```diff
@@ -39,6 +41,16 @@ import { createRouter } from '@backstage/plugin-techdocs-backend';
 export const techdocsPlugin = createBackendPlugin({
   pluginId: 'techdocs',
   register(env) {
+    let docsBuildStrategy: DocsBuildStrategy | undefined;
+    env.registerExtensionPoint(techdocsBuildStrategyExtensionPoint, {
```

Comment:
`techdocsBuildStrategyExtensionPoint` may rather be too _narrow_ an extension point - what are the appropriate domains here? Maybe "building" overall is a domain. So `techdocsBuildsExtensionPoint` or something like that, so more methods can conceivably be added?